keyboards/rubi/lib/glcdfont.c

```diff
@@ -0,0 +1,255 @@
+  0x00, 0x00, 0x00, 0x08, 0x08, 0x3E,
+  0x08, 0x08, 0x00, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x00, 0x08, 0x08, 0x08,
+  0x08, 0x08, 0x00, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x3E, 0x6F, 0x5F, 0x63,
+  0x7B, 0x7B, 0x3E, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x3E, 0x7F, 0x59, 0x6F,
+  0x77, 0x7B, 0x3E, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x00, 0x14, 0x08, 0x3E,
+  0x08, 0x14, 0x00, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x00, 0x08, 0x08, 0x2A,
+  0x08, 0x08, 0x00, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x7E, 0x04, 0x08, 0x10,
+  0x7E, 0x00, 0x3E, 0x40, 0x40, 0x40,
+  0x3E, 0x00, 0x7E, 0x04, 0x08, 0x04,
+  0x7E, 0x00, 0x00, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x3C, 0x42, 0x42, 0x42,
+  0x24, 0x00, 0x7C, 0x12, 0x12, 0x12,
+  0x7C, 0x00, 0x7E, 0x12, 0x12, 0x12,
+  0x0C, 0x00, 0x00, 0x00, 0x00, 0x00,
+  0x7E, 0xFF, 0x81, 0xFB, 0xE7, 0xFB,
+  0x81, 0xFF, 0x81, 0xB5, 0xB5, 0xB5,
+  0xBD, 0xFF, 0x81, 0xBD, 0xBD, 0xBD,
+  0xC3, 0xFF, 0x7E, 0x00, 0x00, 0x00,
+  0x7E, 0xFF, 0x81, 0xB5, 0xB5, 0xB5,
+  0xCB, 0xFF, 0x81, 0xEF, 0xF7, 0xEB,
+  0x9D, 0xFF, 0x81, 0xBF, 0xBF, 0xBF,
+  0xBF, 0xFF, 0x7E, 0x00, 0x00, 0x00,
+  0x7E, 0xFF, 0x81, 0xED, 0xED, 0xED,
+  0xF3, 0xFF, 0x83, 0xED, 0xED, 0xED,
+  0x83, 0xFF, 0x81, 0xBD, 0xBD, 0xBD,
+  0xC3, 0xFF, 0x7E, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x3E, 0x77, 0x6F, 0x77,
+  0x7B, 0x77, 0x3E, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x00, 0x08, 0x10, 0x08,
+  0x04, 0x08, 0x00, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x3E, 0x77, 0x77, 0x55,
+  0x77, 0x77, 0x3E, 0x00, 0x00, 0x00,
+  0x7E, 0xFF, 0xB3, 0xAD, 0xAD, 0xAD,
+  0xDB, 0xFF, 0xC3, 0xBD, 0xBD, 0xBD,
+  0xDB, 0xFF, 0x81, 0xED, 0xCD, 0xAD,
+  0xB3, 0xFF, 0x7E, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x00, 0x10, 0x20, 0x1C,
+  0x04, 0x04, 0x00, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x3C, 0x42, 0x42, 0x42,
+  0x24, 0x00, 0x7C, 0x12, 0x12, 0x12,
+  0x7C, 0x00, 0x7E, 0x40, 0x40, 0x40,
+  0x40, 0x00, 0x00, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x7E, 0x12, 0x12, 0x12,
+  0x0C, 0x00, 0x7C, 0x12, 0x12, 0x12,
+  0x7C, 0x00, 0x7E, 0x42, 0x42, 0x42,
+  0x3C, 0x00, 0x00, 0x00, 0x00, 0x00,
+  0x7E, 0xFF, 0xC3, 0xBD, 0xBD, 0xBD,
+  0xC3, 0xFF, 0x7E, 0x00, 0x00, 0x00,
+  0x7E, 0xFF, 0xFF, 0xBB, 0x81, 0xBF,
+  0xFF, 0xFF, 0x7E, 0x00, 0x00, 0x00,
+  0x7E, 0xFF, 0x9B, 0xAD, 0xAD, 0xAD,
+  0xB3, 0xFF, 0x7E, 0x00, 0x00, 0x00,
+  0x7E, 0xFF, 0xDD, 0xBD, 0xB5, 0xB5,
+  0xC9, 0xFF, 0x7E, 0x00, 0x00, 0x00,
+  0x7E, 0xFF, 0xE7, 0xEB, 0xED, 0x81,
+  0xFF, 0xFF, 0x7E, 0x00, 0x00, 0x00,
+  0x7E, 0xFF, 0xD1, 0xB5, 0xB5, 0xB5,
+  0xCD, 0xFF, 0x7E, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x3E, 0x77, 0x77, 0x41,
+  0x77, 0x77, 0x3E, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x3E, 0x77, 0x77, 0x77,
+  0x77, 0x77, 0x3E, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x3E, 0x6B, 0x77, 0x41,
+  0x77, 0x6B, 0x3E, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x4C, 0x52, 0x52, 0x52,
+  0x24, 0x00, 0x3C, 0x42, 0x42, 0x42,
+  0x24, 0x00, 0x7E, 0x12, 0x32, 0x52,
+  0x4C, 0x00, 0x00, 0x00, 0x00, 0x00,
+  0x00, 0x00, 0x00, 0x00, 0x26, 0x10,
+  0x08, 0x04, 0x00, 0x00, 0x00, 0x00,
+};
```

Comment:
if you need more characters, you can get 22 more.

Reply:
I think this is enough. Actually I was just copied from another board, then modified it to fit my needs.